packages/near-fast-auth-signer/src/utils/string.ts

```diff
@@ -0,0 +1,9 @@
+export function generateRandomString(length: number) {
```

Comment:
I think we have the UUID library installed for this. 

It's also better for collision proof